### DISCUSSION TITLE
Properties in constructor

### DISCUSSION THREAD

**Discussion by minogin** [LemonAppDev/konsist#382] [Ask a Question] [answered]
### Wanted end result.

Having this class
```
class TestController(
    private val testService: TestService
) {}
```
Want to get `testService` as a TestController class property like this:
```
Konsist.classes()
  .withName("TestController")
  .hasProperty {
    it.type.name.startsWith("test")
  }
```

### What you tried so far.

Tried what's above

**Comment by igorwojda** [comment, marked as answer]
This is a bug. It was just fixed in https://github.com/LemonAppDev/konsist/releases/tag/v0.11.0
Thanks for the contribution.
